python/nav/util.py

```diff
@@ -221,6 +221,14 @@ def first_true(iterable, default=None, pred=None):
     return next(ifilter(pred, iterable), default)
 
 
+def chunks(lst, size):
```

Comment:
Would `itertools.tee` be relevant here?

Reply:
Actually, it seems it would be equivalent to the `grouper()` recipe from https://docs.python.org/3/library/itertools.html#itertools-recipes (which doesn't require the input to be a sequence with a determinable length).

Reply:
.. and then I wound up in the "perfect chunker" rabbit-hole... this is fine.